backend/app/services/direct_chat_service.py

```
@@ -0,0 +1,389 @@
+        device_id = payload.device_id
+        if not device_id:
+            raise HTTPException(status_code=400, detail="device_id is required")
+        if not device_service.get_device_by_device_id(db, user.id, device_id):
+            raise HTTPException(status_code=404, detail="Device not found")
+
+        team = (
+            db.query(Kind)
+            .filter(
+                Kind.id == payload.team_id,
+                Kind.kind == "Team",
+                Kind.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)